test/intl402/ListFormat/constructor/supportedLocalesOf/result-type.js

```diff
@@ -21,14 +21,14 @@ assert.sameValue(Object.isExtensible(result), true,
 assert.notSameValue(result.length, 0);
 for (let i = 0; i < result.length; ++i) {
   verifyProperty(result, String(i), {
-    "writable": false,
+    "writable": true,
     "enumerable": true,
-    "configurable": false,
+    "configurable": true,
   });
 }
 
 verifyProperty(result, "length", {
-  "writable": false,
+  "writable": true,
```

Comment:
It looks like `verifyProperty` is too limited to successfully check writability of Array.length, so I'd just drop this line.

Reply:
Thanks. I update the PR. PTAL. Thanks